memphis/consumer.py

```diff
@@ -108,14 +109,17 @@ async def __consume_dls(self):
                         self.context,
                     )
         except Exception as e:
-            await self.dls_callback_func([], MemphisError(str(e)), self.context)
-            return
+            if self.dls_callback_func != None:
```

Comment:
Could you explain why you did it?

Reply:
when you call to Fetch function, you dont pass callback parameter. you can pass only batch_size parameter. there is function calls __consume_dls which run when the sdk start. when i put the raise error of the batch_size in Fetch function, the consume_dls function fall and go the the execpt where the callback is called. but in Fetch function there is not callback function so the code is fail. i put this if to check if there is a callback function to avoid this situation .